arcade/sprite_list/sprite_list.py

```diff
@@ -790,13 +790,15 @@ def disable_spatial_hashing(self) -> None:
         """
         self.spatial_hash = None
 
-    def enable_spatial_hashing(self, spatial_hash_cell_size=128):
+    def enable_spatial_hashing(self, spatial_hash_cell_size: int = 128):
         """Turn on spatial hashing."""
```

Comment:
Docsting should probalby be improved here? Parameters. What happens if you already have a spatial hash?

Reply:
My intent was to do the docstring in a subsequent PR since I'm working on separating the Programming Guide content out of them.